pubsub/cloud-client/publisher.py

```diff
@@ -171,17 +172,17 @@ def callback(f):
             try:
                 print(f.result())
                 futures.pop(data)
-            except: # noqa
+            except:  # noqa
```

Comment:
nit: is this space required by the linter?

Reply:
Yah, double space is required for single line comments.